modules/installation-aws-config-yaml.adoc

```diff
@@ -312,7 +312,12 @@ endif::gov,secret,china[]
 ifdef::gov,secret,china[]
 <1> Required.
 endif::gov,secret,china[]
-<2> Optional: Add this parameter to force the Cloud Credential Operator (CCO) to use the specified mode, instead of having the CCO dynamically try to determine the capabilities of the credentials. For details about CCO modes, see the _Cloud Credential Operator_ entry in the _Red Hat Operators reference_ content.
+<2> Optional: Add this parameter to force the Cloud Credential Operator (CCO) to use the specified mode, instead of having the CCO dynamically try to determine the capabilities of the credentials. For details about CCO modes, see _About the Cloud Credential Operator_ in the _Authentication and authorization_ content.
++
+[IMPORTANT]
```

Comment:
I think readers would be better served if you just told them to set the parameter to manual as part of Clusters configured to use short-term credentials only: Incorporating the Cloud Credential Operator utility files.

This could easily be over looked here. I know these samples have been used in similar ways previously, but I do believe we should be rethinking the level of detail we put in them.